core/species.cpp

```diff
@@ -4395,7 +4967,7 @@ void Species::DerivedStatesFromAscii(tsk_table_collection_t *p_tables)
 										 binary_derived_state_offset.data(),
 										 mutations_copy.metadata,
 										 mutations_copy.metadata_offset);
-		if (ret < 0) handle_error("convert_from_ascii", ret);
+		if (ret < 0) handle_error("derived_to_ascii", ret);
```

Comment:
I think we're in DerivedStatesFromAscii here (not "to")?

Reply:
Ah, huh; I thought I was fixing a typo, but perhaps I introduced one, lol

Reply:
fixed, resolving